applications/asset_tracker/prj.conf

```diff
@@ -90,24 +86,22 @@ CONFIG_HEAP_MEM_POOL_SIZE=16384
 CONFIG_MAIN_STACK_SIZE=8192
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
 
+# Settings
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_FLASH_MAP=y
+CONFIG_FCB=y
+CONFIG_SETTINGS=y
+CONFIG_SETTINGS_FCB=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
+
 # MCUBOOT
 CONFIG_BOOTLOADER_MCUBOOT=y
 CONFIG_IMG_MANAGER=y
 CONFIG_MCUBOOT_IMG_MANAGER=y
-
-# Flash
-CONFIG_FLASH=y
```

Comment:
Nit: Align the changes if possible so it's a bit easier to see. Currently, it looks like you do changes to the flash configuration.

Reply:
It's a nit so feel free to ignore it :)